tokens.js

```diff
@@ -11,21 +15,35 @@ const isTokenExpired = (token) => (Date.now() >= JSON.parse(Buffer.from(token.sp
  */
 async function generateNewAccessToken() {
     try {
-        const { tokenObject } = await got.post(process.env.RERUM_ACCESS_TOKEN_URL, {
+        const tokenObject = await got.post(process.env.RERUM_ACCESS_TOKEN_URL, {
             timeout: 10000,
-            json: { REFRESH_TOKEN: process.env.REFRESH_TOKEN }
+            json: { "refresh_token": process.env.REFRESH_TOKEN }
         }).json()
-
         process.env.ACCESS_TOKEN = tokenObject.access_token
+        try{
+            const data = await fs.readFile('./.env', { encoding: 'utf8' })
+            // Please note that this parse() will remove all #comments in the .env file.
+            let env_file_obj = parse(data)
+            env_file_obj.ACCESS_TOKEN = tokenObject.access_token
+            await fs.writeFile('./.env', stringify(env_file_obj))
+            console.log("TinyNode now has an updated access token.")
```

Comment:
works locally here, I hope the runners can do the same. We don't have a test for this either, which should just test the file access, methinks.